mongodb.md

```diff
@@ -0,0 +1,85 @@
+
+The connection string for the MongoDB server can be set in the `.env` file:
+
+```
+MONGODB_URI=mongodb://localhost:27017
+MONGODB_DATABASE=laravel_app
+```
+
+For hosting MongoDB in the cloud, consider using [MongoDB Atlas](https://www.mongodb.com/cloud/atlas).
+To access a MongoDB Atlas cluster locally from you application, you need to [allow your own IP address in the cluster's network settings](https://www.mongodb.com/docs/atlas/data-federation/tutorial/add-ip-address/).
+
+The connection string for MongoDB Atlas can be set in the `.env` file:
+
+```
+MONGODB_URI=mongodb+srv://<username>:<password>@<cluster>.mongodb.net/<dbname>?retryWrites=true&w=majority
+MONGODB_DATABASE=laravel_app
+```
+
+#### MongoDB Driver
+
+The MongoDB PHP driver requires the `ext-mongodb` PHP extension. The extension can be installed using PECL:
+
+```shell
+pecl install mongodb
+```
+
+Read the [MongoDB PHP driver installation instructions](https://www.php.net/manual/en/mongodb.installation.php) for more information.
+
+#### Install the Laravel MongoDB package
+
+The Laravel MongoDB package can be installed via Composer:
+
+```shell
+composer require mongodb/laravel-mongodb
+```
+
+This installation will fail if the `ext-mongodb` PHP extension is not installed.
+
+<a name="configuration"></a>
+## Configuration
+
+You may configure your MongoDB connection via the `config/database.php` configuration file.
+Within this file, you can add a `mongodb` connection using the `mongodb` driver:
+
+```php
+'connections' => [
+    'mongodb' => [
+        'driver' => 'mongodb',
+        'dsn' => env('MONGODB_URI', 'mongodb://localhost:27017'),
+        'database' => env('MONGODB_DATABASE', 'laravel_app'),
+    ],
+],
+```
+
+Once your configuration is complete, you can use the `mongodb` connection in your application:
+
+- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Using raw SQL queries is not supported obviously, as MongoDB is a NoSQL database.
+- [Write complex queries](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/) using the query builder.
+- [Using the Cache](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/cache/), the `mongodb` cache driver is optimized to use MongoDB features such as TTL indexes to automatically clear expired cache entries.
+- Using the Session with the `cache` session driver.
+- [Using the Queue](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/) with the `mongodb` queue driver.
+- [Storing files in GridFS](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/gridfs/), thanks to an integration of the  [GridFS Adapter for Flysystem](https://flysystem.thephpleague.com/docs/adapter/gridfs/).
```

Comment:
```suggestion
- [Storing files in GridFS](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/gridfs/), by way of the [GridFS Adapter for Flysystem](https://flysystem.thephpleague.com/docs/adapter/gridfs/).
```

"thanks to" sounds more appropriate for a blog post, but I think docs should be more impersonal. Defer to you and @bisht2050, though.

Reply:
I agree with Jeremy.